app/src/main/res/layout/statistics_fragment.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="10dp">
```

Comment:
10dp is a rather unusual value
You might want to take a look at https://www.google.com/design/spec/layout/metrics-keylines.html#metrics-keylines-keylines-spacing